packages/kbn-optimizer/limits.yml

```diff
@@ -46,7 +46,7 @@ pageLoadAssetSize:
   lens: 96624
   licenseManagement: 41817
   licensing: 39008
-  lists: 202261
+  lists: 228500
```

Comment:
This is expected. Looking at the metrics, the security solution was minus about the same amount as was added to the lists plugin. Since the lists plugin previously did not include UI components, it was able to stay pretty small.

Reply:
👍 was curious about that [here](https://github.com/elastic/kibana/pull/95266#discussion_r602031432). Would you mind reducing the securitySolutions limit since that was moved out?